### PR TITLE
upgrades: Documentation improvements

### PR DESCRIPTION
**Pull request by efritz**
Make upgrade docs more explicit.

## Test plan

Docs update.

### CODE REVIEW

File: doc/admin/deploy/kubernetes/update.md [hunk 1]
```diff
@@ -66,7 +71,7 @@ To perform a multi-version upgrade on a Sourcegraph instance running on Kubernet
   - Stateful sets (e.g., `kubectl scale sts <name> --replicas=0`):
       - gitserver
       - indexed-search
-1. Run the `migrator upgrade` command targetting the same databases as your instance. See the [command documentation](./../../how-to/manual_database_migrations.md#upgrade) for additional details.
+1. Run the `migrator upgrade` command targetting the same databases as your instance. See the [command documentation](./../../how-to/manual_database_migrations.md#upgrade) for additional details. In short, the migrator is invoked as a Kubernetes job (a short-lived container) using the same Kubernetes cluster and using environment variables indicating the instance's databases.
```
Comment (DaedalusG) [inline doc/admin/deploy/kubernetes/update.md:74]:
```suggestion
1. Run the `migrator upgrade` command targetting the same databases as your instance. See the [command documentation](./../../how-to/manual_database_migrations.md#upgrade) for additional details. In short, the migrator is invoked as a [Kubernetes job](https://github.com/sourcegraph/deploy-sourcegraph/blob/master/configure/migrator/migrator.Job.yaml) (a short-lived container) using the same Kubernetes cluster and using environment variables indicating the instance's databases.
```

File: doc/admin/deploy/kubernetes/helm.md [hunk 2]
```diff
@@ -933,7 +943,7 @@ To perform a multi-version upgrade on a Sourcegraph instance running on Kubernet
   - Stateful sets (e.g., `kubectl scale sts <name> --replicas=0`):
       - gitserver
       - indexed-search
-1. Run the `migrator upgrade` command targetting the same databases as your instance. See the [command documentation](./../../how-to/manual_database_migrations.md#upgrade) for additional details.
+1. Run the `migrator upgrade` command targetting the same databases as your instance. See the [command documentation](./../../how-to/manual_database_migrations.md#upgrade) for additional details. In short, the migrator is invoked as a Kubernetes job (a short-lived container) using the same Kubernetes cluster and using environment variables indicating the instance's databases.
```
Comment (DaedalusG) [inline doc/admin/deploy/kubernetes/helm.md:946]:
```suggestion
1. Run the `migrator upgrade` command targetting the same databases as your instance. See the [command documentation](./../../how-to/manual_database_migrations.md#upgrade) for additional details. In short, the migrator is invoked as a [Kubernetes job](https://github.com/sourcegraph/deploy-sourcegraph/blob/master/configure/migrator/migrator.Job.yaml) (a short-lived container) using the same Kubernetes cluster and using environment variables indicating the instance's databases.
```

File: doc/admin/deploy/docker-compose/upgrade.md [hunk 3]
```diff
@@ -69,5 +75,5 @@ To perform a multi-version upgrade on a Sourcegraph instance running on Docker c
   - `docker-compose up -d pgsql`
   - `docker-compose up -d codeintel-db`
   - `docker-compose up -d codeinsights-db`
-1. Run the `migrator upgrade` command targetting the same databases as your instance. See the [command documentation](./../../how-to/manual_database_migrations.md#upgrade) for additional details.
+1. Run the `migrator upgrade` command targetting the same databases as your instance. See the [command documentation](./../../how-to/manual_database_migrations.md#upgrade) for additional details. In short, the migrator is invoked with a `docker run` command using the same compose network and using environment variables indicating the instance's databases.
```
Comment (DaedalusG) [inline doc/admin/deploy/docker-compose/upgrade.md:78]:
```suggestion
1. Run the `migrator upgrade` command targetting the same databases as your instance. See the [command documentation](./../../how-to/manual_database_migrations.md#upgrade) for additional details. In short, the [migrator](https://github.com/sourcegraph/deploy-sourcegraph-docker/blob/master/docker-compose/docker-compose.yaml#L3-L53) is invoked with a `docker run` command using the same compose network and using environment variables indicating the instance's databases.
```

**Review by DaedalusG — APPROVED** [review]
Looks great -- added some hyperlinks to clarify _exactly_ where the migrator env vars are at and whats being invoked by `docker run` / kubectl

File: doc/admin/deploy/docker-compose/upgrade.md (hunk 3 above)
Comment (efritz) [reply doc/admin/deploy/docker-compose/upgrade.md:78]:
```suggestion
1. Run the `migrator upgrade` command targetting the same databases as your instance. See the [command documentation](./../../how-to/manual_database_migrations.md#upgrade) for additional details. In short, the [migrator](https://sourcegraph.com/search?q=context:global+repo:%5Egithub%5C.com/sourcegraph/deploy-sourcegraph-docker%24+file:docker-compose%5C.yaml+container_name:+migrator) is invoked with a `docker run` command using the same compose network and using environment variables indicating the instance's databases.
```

For a more stable URL.

**Pull request merged**